### PR TITLE
Parallel extract merge

### PR DESCRIPTION
**Pull request by macintux**
Add tests for 2i parallel extract and return_body.

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -11,11 +11,12 @@
 {eunit_opts, [verbose]}.
 
 {deps, [
+        {riak_pb, ".*", {git, "git://github.com/basho/riak_pb", {branch, "feature/jrd/index_body"}}},
```
Comment (mrallen1) [inline rebar.config]: Just want to make sure these are the right branches for the dependencies.

File: rebar.config [hunk 2]
```diff
@@ -11,11 +11,12 @@
...
+        {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "feature/jrd/index_body"}}},
```
Comment (mrallen1) [inline rebar.config]: We talked about some other branches in HipChat :)

Comment (macintux) [inline rebar.config]: Yep, I didn’t review rebar.config changes, will have to tweak these, thanks.

-John

**Comment by mrallen1** [comment]
Looks good here. Ran these tests for 6+ hours without issue.

**Comment by mrallen1** [comment]
:+1: f2cae65

**Pull request merged**